code/datums/station_traits/neutral_traits.dm

```diff
@@ -189,3 +189,101 @@
 		return
 
 	gorilla.poll_for_gorilla()
+
+/datum/station_trait/birthday
+	name = "Employee Birthday"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 2
+	show_in_report = TRUE
+	report_message = ""
```

Comment:
why are you showing it in report if theres no message?

Reply:
That would be because I forgot :3c